dcpy/lifecycle/ingest/templates/bpl_libraries.yml

```diff
@@ -6,14 +6,13 @@ source:
   format: json
 file_format:
   type: json
-  normalize:
-    record_path:
-    - locations
+  json_read_fn: normalize
+  json_read_kwargs: { "record_path": ["locations"] }
 processing_steps:
-- name: clean_column_names
-  args:
-    replace: 
-      "data.": ""
+  - name: clean_column_names
+    args:
+      replace:
+        "data.": ""
```

Comment:
Are you using a yml linter/formatter? And if so which one 😬

Reply:
I use this vscode yaml extension: 

<img width="1077" alt="image" src="https://github.com/NYCPlanning/data-engineering/assets/144725249/02faf761-451f-4ab0-904a-7e48cdb8d16b">


cc @damonmcc